content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -419,6 +419,43 @@ securityContext:
     localhostProfile: my-profiles/profile-allow.json
 ```
 
+## Set the AppArmor Profile for a Container
+
+To set the AppArmor profile for a Container, include the `appArmorProfile` field
+in the `securityContext` section of your Pod or Container manifest. The
+`appArmorProfile` field is a
+[AppArmorProfile](/docs/reference/generated/kubernetes-api/{{< param "version"
+>}}/#apparmorprofile-v1-core) object consisting of `type` and `localhostProfile`.
+Valid options for `type` include `RuntimeDefault`, `Unconfined`, and
+`Localhost`. `localhostProfile` must only be set if `type: Localhost`. It
+indicates the name of the pre-configured profile on the node. The profile needs
+to loaded onto all nodes, since you don't know where the pod will be scheduled. 
+Approaches of setting up custom profiles are discussed in
+[Setting up nodes with profiles](/docs/tutorials/security/apparmor/#setting-up-nodes-with-profiles). 
+
+Here is an example that sets the AppArmor profile to the node's container runtime
+default profile:
+
+```yaml
+...
```

Comment:
make sure it is clear that this is a container's securityContext, not the Pod's

Reply:
Done.